packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -13,7 +13,7 @@
 </template>
 
 <script lang="ts">
-import ShowDetails from '../../mixins/showDetails'
+import { useActionsShowDetails } from '../../../../web-pkg/src/composables/actions'
```

Comment:
Relative import from `web-pkg` 🙈 In general we can combine the imports from `web-pkg/src/composables/actions` into one here.